algorithms/aqc/solving_qlsp/solving_vqls_with_aqc.metadata.json

```diff
@@ -0,0 +1,7 @@
+{
+  "friendly_name": "Solving QLSP wuth AQC - Part 1",
```

Comment:
*with - typo

Reply:
Also I wouldn't call it 'Part 1'. In the IDE I would give it a stand-alone name. Maybe 'naive' or 'linear scheduling' or 'trotterization' etc